src/amo/components/Addon/index.js

```diff
@@ -253,11 +253,22 @@ export class AddonBase extends React.Component {
 export function mapStateToProps(state, ownProps) {
   const { slug } = ownProps.params;
   const addon = state.addons[slug];
-  const installation = state.installations[addon.guid];
+  const installedAddon = state.installations[addon.guid] || {};
 
   return {
     addon,
-    installStatus: installation ? installation.status : UNKNOWN,
+    // These spreads obscure a lot of hidden properties but they
```

Comment:
Well, I'm sure that's just one of many regressions that could occur! I don't know how helpful it would be. There is a lot of theme interaction that would break too.